walletlibrarydemo/src/main/java/com/microsoft/walletlibrarydemo/feature/issuance/viewlogic/LoadRequestFragment.kt

```diff
@@ -0,0 +1,80 @@
+package com.microsoft.walletlibrarydemo.feature.issuance.viewlogic
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.microsoft.walletlibrary.requests.requirements.VerifiedIdRequirement
+import com.microsoft.walletlibrary.verifiedid.VerifiedId
+import com.microsoft.walletlibrarydemo.databinding.LoadRequestFragmentBinding
+import com.microsoft.walletlibrarydemo.feature.issuance.presentationlogic.ClickListener
+import com.microsoft.walletlibrarydemo.feature.issuance.presentationlogic.SampleViewModel
+import com.microsoft.walletlibrarydemo.feature.issuance.presentationlogic.VerifiedIdsAdapter
+import com.microsoft.walletlibrarydemo.feature.issuance.presentationlogic.ViewModelFactory
+import kotlinx.coroutines.runBlocking
+
+class LoadRequestFragment : Fragment(), ClickListener {
+    private var _binding: LoadRequestFragmentBinding? = null
+    private val binding get() = _binding!!
+
+    private val viewModel: SampleViewModel by viewModels {
+        ViewModelFactory(requireContext())
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = LoadRequestFragmentBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        configureViews()
+    }
+
+    private fun configureViews() {
+        binding.createRequest.setOnClickListener { initiateIssuance() }
+        runBlocking {
+            val verifiedIds = viewModel.getVerifiedIds()
```

Comment:
what is this Fragment doing?

Reply:
It only gets the request. I started off with having separate fragments for each functionality but combined everything into one. I will see if I can move this also there.

Reply:
This also displays all the VerifiedIds issued so far in this page. I would keep it as two fragments.